README.md

```diff
@@ -41,7 +42,19 @@ The toolbox currently contains the following tools:
   
 ### Documentation
 
-The documentation, including installation instructions, is available at https://qiskit-extensions.github.io/circuit-knitting-toolbox/.
+All CKT documentation is available at https://qiskit-extensions.github.io/circuit-knitting-toolbox/.
+
+----------------------------------------------------------------------------------------------------
+  
+### Installation
+
+We encourage installing CKT via ``pip``, when possible. Users intending to use the entanglement forging tool should install the ``pyscf`` optional dependency. Users intending to use the automatic cut finding functionality in the ``CutQC`` package should install the ``cplex`` optional dependency.
+
+```bash
+pip install circuit-knitting-toolbox[cplex,pyscf]
```

Comment:
```suggestion
pip install 'circuit-knitting-toolbox[cplex,pyscf]'
```
this is more likely to work across different shells (the brackets can throw off some of them)